src/main/webapp/scripts/common.js

```diff
@@ -27,3 +27,11 @@ function toTopFunction() {
   document.body.scrollTop = 0;
   document.documentElement.scrollTop = 0;
 }
+
+// Fetch blobstore URL, i.e. action URL which is called when form is submitted
+// this URL is needed to enable uploading image to blobstore
+function loadFormSubmissionUrl() {
```

Comment:
The loaded URL is only for the profile picture not for the whole form, right?

Reply:
/blobstore-url (now /upload-url) is created to upload the picture, while also fetching /user-data (in the conversation above) thus this url leads to the whole form submission

Reply:
Ah got it! Thanks for the clarification :)